pyproject.toml

```diff
@@ -58,4 +147,27 @@ max-complexity = 20
 "test/**" = ["D103"]
 "docs/**" = ["D103"]
 "examples/**" = ["D103"]
-"setup.py" = ["D103"]
+
+
+[tool.poetry_bumpversion.file."tianshou/__init__.py"]
+
+[tool.poetry-sort]
+move-optionals-to-bottom = true
+
+# keep relevant parts in sync with pre-commit
+[tool.poe.tasks]  # https://github.com/nat-n/poethepoet
+test = "pytest test --cov=tianshou --cov-report=xml --cov-report=term-missing --durations=0 -v --color=yes"
+test-reduced = "pytest test/base test/continuous --cov=tianshou --durations=0 -v --color=yes"
+_black_check = "black --check ."
+_ruff_check = "ruff check ."
+_black_format = "black ."
+_ruff_format = "ruff --fix ."
+lint = ["_black_check", "_ruff_check"]
+_poetry_install_sort_plugin = "poetry self add poetry-plugin-sort"
+_poery_sort = "poetry sort"
+format = ["_black_format", "_ruff_format", "_poetry_install_sort_plugin", "_poery_sort"]
+_spellcheck = "sphinx-build -W -b spelling docs docs/_build"
+_doc_build = "sphinx-build -W -b html docs docs/_build"
+doc-clean = "rm -rf docs/_build"
+doc-build = ["_spellcheck", "_doc_build"]
+type-check = "mypy tianshou"
```

Comment:
could you also update the corresponding parts in `docs/`?

Reply:
Done. I haven't found any other part of docs that needs updating